package ExeptionPackage;

public class ExampleThree {
    public static void main(String[] args) {
    try{
        int[] arr = {1,2,3,4,5};
    }
    catch (RuntimeException e ){
        System.out.println("run something");
    }}

}
